### PR TITLE
Ui sliding panel

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

**Comment by safchain** [comment]
rerun scale-tests

File: statics/js/components/topology.js [hunk 1]
```diff
@@ -314,21 +328,89 @@ var TopologyComponent = {
...
 
     endsWith: function (str, suffix) {
        return str.indexOf(suffix, str.length - suffix.length) !== -1;
     },
 
+    setFilterFromConfig: function() {
+      return $.when(this.$getConfigValue('analyzer.topology_filter'))
+       .then(function(filter) {
+         var options = $("#topology-filter-list");
+         $.each(filter, function(key, value) {
```
Comment (masco-enovance) [inline statics/js/components/topology.js]: if there is no filter in config file, getting null reference error in jquery lib

Comment (safchain) [reply statics/js/components/topology.js]: Thanks for reporting this issue, fixed

**Comment by masco-enovance** [comment]
rerun scale-tests

File: statics/js/components/topology.js [hunk 2]
```diff
@@ -9,38 +9,81 @@ var TopologyComponent = {
...
-	              </button>\
-	          </div>\
-	        </div> \
-         <div class="topology-controls">\
+        <div class="topology-d3">\
+          <div class="topology-legend">\
+            <strong>Topology view</strong></br>\
+            <span v-if="currTopologyFilter">{{currTopologyFilter}}</span>\
+            <span v-if="!currTopologyFilter">Full</span>\
+            <div v-if="topologyHumanTimeContext">{{topologyHumanTimeContext}}</div>\
```
Comment (masco-enovance) [inline statics/js/components/topology.js]: just a thought: how about having this <div v-else>Live</div>\

File: statics/js/components/topology.js [hunk 3]
```diff
@@ -9,38 +9,81 @@ var TopologyComponent = {
...
-	                <span class="glyphicon glyphicon-search" aria-hidden="true"></span>\
-	              </button>\
-	          </div>\
-	        </div> \
-         <div class="topology-controls">\
+        <div class="topology-d3">\
+          <div class="topology-legend">\
+            <strong>Topology view</strong></br>\
+            <span v-if="currTopologyFilter">{{currTopologyFilter}}</span>\
+            <span v-if="!currTopologyFilter">Full</span>\
```
Comment (masco-enovance) [inline statics/js/components/topology.js]: we can use "v-else"

File: statics/js/components/topology.js (hunk 2 above)
Comment (masco-enovance) [reply statics/js/components/topology.js]: not able to add my code pice here, displaying "Live" in else block is my idea.

**Comment by safchain** [comment]
rerun functional-tests-backend-orientdb

**Comment by masco-enovance** [comment]
run functional-tests-backend-orientdb

**Pull request merged**